readthedocs/core/views/hooks.py

```diff
@@ -183,11 +187,14 @@ def delete_external_version(project, identifier, verbose_name):
     return None
 
 
-def build_external_version(project, version, commit):
+def build_external_version(project, version, version_data):
```

Comment:
version_data isn't used here right now, but it will be used to match external versions.

Reply:
This should be a comment in the code :)